cargo-concordium/src/build.rs

```diff
@@ -422,7 +426,18 @@ pub fn build_and_run_wasm_test(extra_args: &[String]) -> anyhow::Result<bool> {
 
     eprintln!("\n{}", Color::Green.bold().paint("Running tests ..."));
 
-    let results = utils::run_module_tests(&wasm)?;
+    let seed_u64 = match seed {
+        Some(s) => s,
+        None => {
+            // We cannot use `SmallRng::from_entropy()` directly, because is returns an
+            // instance of a RNG and it is not possible to obtain the seed from this
+            // instance. Instead, we generate the seed explicitly and then
+            // instantiate a RNG with it.
```

Comment:
I am not sure I understand this comment with respect to the code below. Is it a leftover?

Reply:
Updated.